server/index.js

```diff
@@ -1,27 +1,19 @@
 require('dotenv/config');
 const express = require('express');
 const pg = require('pg');
-// eslint-disable-next-line no-unused-vars
-const ClientError = require('./client-error');
-const errorMiddleware = require('./error-middleware');
-const staticMiddleware = require('./static-middleware');
-const pathToSwaggerUi = require('swagger-ui-dist').absolutePath();
-const cors = require('cors');
 const db = new pg.Pool({
+  const swaggerUi = require('swagger-ui-express');
+  const YAML = require('yamljs');
```

Comment:
These `require()` calls look out-of-place. I'd imagine that this crashes the server.